### PR TITLE
Allow passing additional data to getAccessToken()

### PR DESCRIPTION
**Pull request by HSAR**
This enables scenarios including the use of this library to authenticate with Azure Active Directory.

### CODE REVIEW

**Comment by HSAR** [comment]
A proposed implementation of #13.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-oauth2",
-  "version": "2.3.1",
+  "version": "2.3.2",
```
Comment (mawie81) [inline package.json]: Please leave at 2.3.1. I will update the version when doing  the next release (which would be 2.4.0)

File: index.js [hunk 2]
```diff
@@ -99,11 +99,13 @@ module.exports = function (config, windowParams) {
   function getAccessToken(opts) {
     return getAuthorizationCode(opts)
       .then(authorizationCode => {
-        return tokenRequest({
+        var tokenRequestData = {
           code: authorizationCode,
           grant_type: 'authorization_code',
           redirect_uri: config.redirectUri
-        });
+        }
```
Comment (mawie81) [inline index.js]: There is a semicolon missing here

**Comment by HSAR** [comment]
Your issues should be addressed, and I've updated the readme to reflect the new option :)

**Comment by mawie81** [comment]
Just published as 2.4.0 😄

**Pull request merged**